### PR TITLE
ProbABEL v0.5.0

### PR DESCRIPTION
**Pull request by rtripath89**
Created an EasyConfig for ProbABEL v0.5.0

### CODE REVIEW

File: easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    'files': ['bin/probabel', 'bin/pacoxph',
+	'bin/palinear', 'bin/palogist','etc/probabel_config.cfg.example'],
+    'dirs': ['share'],
+}
+
+moduleclass = 'bio'
+
+## You need to change the file /etc/probabel_config.cfg.example
+## to etc/probabel_config.cfg in the install directory for
+## it to work.
```
Comment (boegel) [inline easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb]: @rtripath89 can you clarify this?

is it just copying `etc/probabel_config.cfg.example` to `etc/probabel_config.cfg`, or does the file need to be edited as well?

if so, what needs to be changed? what happens if this is not done?

Comment (rtripath89) [inline easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb]: @boegel : It's just renaming the file, Kenneth.
If you don't do it, running the executables gives an error saying you forgot to rename the file, and won't work. Is there a way I can do the renaming in the Easyconfig itself, so we don't have to leave it to the users to change it manually?

Comment (boegel) [inline easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb]: Sure, you can use `postinstallcmds`:

```python
postinstallcmds = ["mv %(installdir)s/etc/probabel_config.cfg.example %(installdir)s/etc/probabel_config.cfg"]
```

I would put this above `sanity_check_paths`, with an empty line in between (order doesn't matter in an easyconfig file, it's just to adhere to the "common stye" we maintain in easyconfig files)

**Comment by boegel** [comment]
@rtripath89 ping?

**Comment by rtripath89** [comment]
Ahhh...sorry, I guess I missed the mail about your comment ... I'll push the change right away!!!! :)

File: easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ('Eigen', '3.2.6')
+]
+
+configopts = '--disable-latex-doc'
+
+postinstallcmds = ["mv %(installdir)s/etc/probabel_config.cfg.example %(installdir)s/etc/probabel_config.cfg"]
+
+sanity_check_paths = {
+    'files': ['bin/probabel', 'bin/pacoxph',
+	'bin/palinear', 'bin/palogist','etc/probabel_config.cfg.example'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb]: @rtripath89 file was renamed, so last entry here should be `etc/probabel_config.cfg` to make the sanity check pass (which probably means you can collapse this line with the line above as well)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1ed7faf3b1a77fb9420f884dc8508d61 for a full test report.

**Comment by rtripath89** [comment]
Updated the Easyconfig. @boegel : I had a question not related to this Easyconfig. I am trying to install Amber14, but it has been more then 16 hours and it seems stuck at install step. It hasn't thrown up any errors, and seems to be still running. I was wondering if you know how much time does it take for Amber's installation via Easconfig, as this seems like a really long time.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5f1c9591240f5cfe42d3b23a849b7fa5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2459.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c9bd3ef1f393ce5070379d31ab0073de for a full test report.

**Comment by boegel** [comment]
@rtripath89 Amber14 takes about 2 hours, maybe it's stuck running tests or something? I suggest you open a separate issue for this.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @rtripath89!

**Pull request merged**